introduction.tex

```diff
@@ -1,15 +1,31 @@
 \section{Introduction} \label{sec:intro}
 
-The current situation is  that, only the Science Pipelines is released as follows:
+This document collects the needs in therms of releses from different stakehoders, section \ref{sec:reqs}.
+These needs are presented as requirements, that shall not be confused with project requirements, like for example the \gls{DMS} requirement specification, documented in \citeds{LSE-61}.
```

Comment:
This document presents the requirements for \gls{LSST} software releases from the various stakeholders. These requirements, detailed in section \ref{sec:reqs}, are not formal project requirements, as given for example in the \gls{DMS} requirements specification \citeds{LSE-61}.